Classes/OldModel/Event.php

```diff
@@ -2712,7 +2712,7 @@ protected function canViewRegistrationsListForWorldAccess(
      * @return string an empty string if everything is OK, a localized error
      *                error message otherwise
      */
-    public function canViewRegistrationsListMessage($whichPlugin, $accessLevel = 'attendees_and_managers'): string
+    public function canViewRegistrationsListMessage(string $whichPlugin, $accessLevel = 'attendees_and_managers'): string
```

Comment:
`$accesslevel` is a `string`.

Reply:
True, its a string but it passed as 2nd param to `canViewRegistrationsList($whichPlugin, $accessLevel)` but the 2nd param is an integer

```    
public function canViewRegistrationsList(
        string $whichPlugin,
        int $registrationsListPID = 0,
        int $registrationsVipListPID = 0,
        int $defaultEventVipsFeGroupID = 0,
        string $accessLevel = 'attendees_and_managers'
    ):
```

Reply:
Thanks!